pkg/didcomm/protocol/outofband/models.go

```diff
@@ -27,5 +27,5 @@ type Invitation struct {
 	Goal      string        `json:"goal,omitempty"`
 	GoalCode  string        `json:"goal-code,omitempty"`
```

Comment:
Shouldn't this be json tag `goal_code`
https://github.com/hyperledger/aries-rfcs/blob/master/features/0434-outofband/README.md#messages

Reply:
Fixed